website/src/components/IndexPage/getOrganismStatistics.ts

```diff
@@ -49,7 +50,7 @@ const getTotalAndLastUpdatedAt = async (
     const client = LapisClient.createForOrganism(organism);
     return (
         await client.call('aggregated', {
-            version: 1,
+            [VERSION_STATUS_FIELD]: siloVersionStatuses.latestVersion,
```

Comment:
🤔  good point. I also can't come up with a good way. IMO this change would still be preferable to no change:
- with this change there is a potential edge case, most likely in the first 30 days, where as you say - this could look inconsisent (if we wrote `X submissions in the past month` it would be OK, but the `+` would look odd)
- _without_ this change, as soon as there is a revocation there is a permanent inconsistency in which when you click on an organism the search page shows a different number of total sequences to the index page

Reply:
All we'd have to do is subtract the recently added sequences that are revoked here?
https://github.com/loculus-project/loculus/blob/e5ad697d2572fa14849efe41d43538a169bdc77f/website/src/components/IndexPage/getOrganismStatistics.ts#L72-L75

Agreed the small chance of inconsistency does not make the big improvement not woth it.

Reply:
> All we'd have to do is subtract here the recently added sequences that are revoked?

I forget what happens if a sequence is revised and then revoked

Reply:
we just:
1. get all sequences that have latest version being revoked
2. filter on release data of their _first_ version being within 1 month
3. count